packages/sdk/contractkit/README.md

```diff
@@ -28,7 +28,7 @@ npm install @celo/contractkit
 yarn add @celo/contractkit
 ```
 
-You will need Node.js v12.x.
+You will need Node.js v12.x. or greater
```

Comment:
contractkit works with node 16 now?

Reply:
the use-contractkit tests run on node 12, 14, and LTS https://github.com/celo-org/use-contractkit/runs/5673083971?check_suite_focus=true

Since `use-contractkit` includes ContractKit I think its safe to say so.